packages/commerce-sdk-react/setup-jest.js

```diff
@@ -39,6 +39,9 @@ Object.defineProperty(window, 'localStorage', {
     value: localStorageMock
 })
 
-global.beforeEach(() => {
+global.afterEach(() => {
+    nock.cleanAll()
```

Comment:
We need to call `cleanAll` especially because we're [persisting](https://github.com/nock/nock#persist) the auth mocks. If we don't clean them up, they will accumulate over time.

We've been lucky that we haven't needed to clean them up ourselves, since nockBack automatically does the clean up for us. 

We can verify these by calling `nock.activeMocks()` before and after this line. After setting the mode, the list of active mocks becomes empty. https://github.com/SalesforceCommerceCloud/pwa-kit/blob/f2f3403f9c0339998f5c6597975cd89dc7ce4208/packages/commerce-sdk-react/src/test-utils.tsx#L142

To avoid any future headache, let's also call `cleanAll`.